postgres_driver.go

```diff
@@ -0,0 +1,169 @@
+package es
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+const createTable = `
+	CREATE TABLE %s (
+		-- TODO: Author VARCHAR(255) NOT NULL,
+		ID               BIGSERIAL PRIMARY KEY,
+		Created          TIMESTAMPTZ DEFAULT now() NOT NULL,
+		AggregateID      UUID NOT NULL,
+		AggregateVersion INT NOT NULL,
+		AggregateType    VARCHAR(255) NOT NULL,
+		Type             VARCHAR(255) NOT NULL,
+		Payload          JSON NOT NULL,
+
+		CONSTRAINT OptimisticLocking UNIQUE (AggregateID, AggregateVersion)
+	)
```

Comment:
Although the column names are case insensitive (if unquoted), I opted by having them reflect the exat same attribute names we have. This would simplify grepping and any eventual renaming.